src/airflow/dags/configs/dag.yaml

```diff
@@ -1,24 +1,20 @@
-- id: "gene_index"
-- id: "gwas_catalog"
-- id: "variant_index"
-  prerequisites:
-    - "gwas_catalog"
-- id: "v2g"
-  prerequisites:
-    - "variant_index"
-    - "gene_index"
-- id: "finngen"
-- id: "ukbiobank"
-- id: "study_locus_overlap"
-  prerequisites:
-    - "gwas_catalog"
-    - "finngen"
-    - "ukbiobank"
-- id: "locus_to_gene"
-  prerequisites:
-    - "gwas_catalog"
-    - "finngen"
-    - "ukbiobank"
-    - "variant_index"
-    - "v2g"
-    - "study_locus_overlap"
+- id: "variant_annotation"
+# - id: "gene_index"
+# - id: "gwas_catalog"
+# - id: "variant_index"
+#   prerequisites:
+#     - "gwas_catalog"
+# - id: "v2g"
+#   prerequisites:
+#     - "variant_index"
+#     - "gene_index"
+# - id: "ukbiobank"
+# - id: "study_locus_overlap"
+#   prerequisites:
+#     - "gwas_catalog"
+#     - "ukbiobank"
+# - id: "locus_to_gene"
+#   prerequisites:
+#     - "variant_index"
+#     - "v2g"
+#     - "study_locus_overlap"
```

Comment:
I'm not sure if this change is intended?

Reply:
Thank you! Now fixed in https://github.com/opentargets/genetics_etl_python/pull/237/commits/d651a033e1ebd5df6ed6031279c54bc2ae64b42d